src/SnackSession.js

```diff
@@ -292,6 +302,99 @@ export default class SnackSession {
     return url;
   };
 
+  /**
+   * Generates a default app.json template for the snack.
+   * @returns {Object} the app.json template
+   * @function
+   */
+  generateAppJson = (): Object => {
+    const appJsonTemplate = {
+      expo: {
+        slug: this.slug,
+        name: this.name,
+        description: this.description,
+        privacy: 'unlisted',
+        sdkVersion: this.sdkVersion,
+        version: '1.0.0',
+        orientation: 'portrait',
+        primaryColor: '#cccccc',
+        icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+        loading: {
+          icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+          hideExponentText: false,
+        },
+        packagerOpts: {
+          assetExts: ['ttf', 'mp4', 'otf'],
+        },
+        ios: {
+          supportsTablet: true,
+        },
+        android: {
+          package: "org.example.test",
+          permissions: ["CAMERA"],
+          versionCode: 1,
+        },
+      },
+    };
+    return appJsonTemplate;
+  };
+
+  /**
+   * Builds an apk from the snack and returns a url to download the apk.
+   * @param 
+   * @returns {Promise.<string>} A promise that contains the url when fulfilled.
+   * @function
+   */
+  getApkUrlAsync = async (
+    appJson : Object
+  ): Promise<string> => {
+    const opts = {
+      platform : 'android',
+      mode: 'create',
```

Comment:
what happens if this is called while a build job is already in progress?

Reply:
An error.